data/towns.js

```diff
@@ -39,7 +39,7 @@ var towns = {
 { "type": "Feature", "properties": { "Name": "Lira", "Class": "Municipality", "District_H": "District HQ" }, "geometry": { "type": "Point", "coordinates": [ 32.900633638375417, 2.248141897738711 ] } },
 { "type": "Feature", "properties": { "Name": "Luwero", "Class": "Major Town", "District_H": "District HQ" }, "geometry": { "type": "Point", "coordinates": [ 32.499405214837765, 0.833232409955782 ] } },
 { "type": "Feature", "properties": { "Name": "Lyantonde", "Class": "Small Town", "District_H": "District HQ" }, "geometry": { "type": "Point", "coordinates": [ 31.155273847242231, -0.405808860425562 ] } },
-{ "type": "Feature", "properties": { "Name": "Malaba", "Class": "Small Town", "District_H": "District HQ" }, "geometry": { "type": "Point", "coordinates": [ 34.260627980077416, 0.644310206250287 ] } },
+{ "type": "Feature", "properties": { "Name": "Malaba.", "Class": "Small Town", "District_H": "District HQ" }, "geometry": { "type": "Point", "coordinates": [ 34.260627980077416, 0.644310206250287 ] } },
```

Comment:
Those 2 names are the same as the border points in the sheet so they get the same red circles style as the one for borders so putting . at the end, differentiates them because they're  towns